src/ExternalSearch.Providers.Bregg/Constants.cs

```diff
@@ -7,7 +7,17 @@
 namespace CluedIn.ExternalSearch.Providers.Bregg
 {
     public static class Constants
-    {
+    {
+        public struct KeyName
+        {
+            public const string AcceptedEntityType = "acceptedEntityType";
```

Comment:
the old nameof(...) would have returned the value AcceptedEntityType not the camel case that this new version returns. Have you tested if this will cause an issue for a customer upgrading?

Reply:
tested, it's case insensitive